src/bahmni-helm/templates/apps/openmrs/openmrs-deployment.yml

```diff
@@ -86,7 +86,16 @@ spec:
             - name: {{ .Release.Name }}-data
               subPath: "openmrs/configuration_checksums"
               mountPath: "/usr/local/tomcat/.OpenMRS/configuration_checksums/"
-            - name: haiti-data
+            - name: {{ .Release.Name }}-data
+              subPath: "openmrs/lucene"
+              mountPath: "/usr/local/tomcat/.OpenMRS/lucene/"
```

Comment:
Would you know what's in this directory?

Reply:
This contains the lucene index files. If we don't put it in a volume we will have to reindex the db if the container is restarted.